### PR TITLE
Externals and bundled argon2

### PR DESCRIPTION
**Pull request by dylanlt**
https://app.clickup.com/t/2704425/ID-5960
A build error was introduced with argon2: 
`'No loader is configured for ".wasm" files: ../../node_modules/.pnpm/argon2-browser@1.18.0/node_modules/argon2-browser/dist/argon2.wasm'`

This fixes that error and declares some packages as externals to fix a different build error. A build test workflow has been added, too. This build test can be expanded later to include other packages such as admin-ui-components.

### CODE REVIEW

File: .github/workflows/build-tests.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: Auth Build Tests
```
Comment (steve-keep) [inline .github/workflows/build-tests.yml:1]: We can call this `Build Tests` and keep adding to it for any build-related tests.

File: .github/workflows/build-tests.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  auth-build-tests:
```
Comment (steve-keep) [inline .github/workflows/build-tests.yml:8]: same here drop the word `auth` and below in the step 👍

File: src/packages/auth/src/authentication/resolvers/password/resolver.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { BackendProvider, Resolver } from '@exogee/graphweaver';
-import * as argon2 from 'argon2-browser';
+import * as argon2 from 'argon2-browser/dist/argon2-bundled.min.js';
```
Comment (steve-keep) [inline src/packages/auth/src/authentication/resolvers/password/resolver.ts:2]: This import is unusual, can you add a comment above this line to explain why we are importing the file directly?

**Review by steve-keep — COMMENTED** [review]
Overall this looks good, just a few tiny comments.

File: .github/workflows/build-tests.yml (hunk 1 above)
Comment (dylanlt) [reply .github/workflows/build-tests.yml:1]: Yes, missed that one!

File: src/packages/auth/src/authentication/resolvers/password/resolver.ts (hunk 3 above)
Comment (dylanlt) [reply src/packages/auth/src/authentication/resolvers/password/resolver.ts:5]: Good pick-up. There was a lot of context here that is now captured.

**Review by steve-keep — APPROVED** [review]

**Pull request merged**